### ISSUE TITLE
Xhtml file does not support zoom

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#363]
**Test device**
Dirac
**Operating System**
MacOS 10.12.6
**Browser**
Safari 12.1.2
**Problem description**
For https://github.com/phetsims/QA/issues/513
Zooming doesn't seem to work on the xhtml file in safari. Tried using the command++/-. The sim would twitch, but not zoom in at all.

**Comment by KatieWoe** [comment]
Seems to happen on Chrome as well.

**Comment by jessegreenberg** [comment]
Interesting, thanks. Zoom works correctly when using mouse/wheel. And command + +/- works if focus is within the simulation. But if there is no focus then it doesn't work.

**Comment by jessegreenberg** [comment]
In HTML, this fires: `document.body.addEventListener( 'keydown', () => console.log( 'heyo!' ) )`. But in XHTML I never see the output. So adding listeners to the body is problematic in XHTML?

**Comment by jessegreenberg** [comment]
Changing KeyStateTracker to attach listeners to window instead of body seems to fix things and still work in the HTML, but I don't understand why it isn't working in the HTML in the first place. I might end up doing this, but want to think about it a little more.

**Comment by jessegreenberg** [comment]
It looks like listeners attached to the `document` are also correctly called in XHTML. Not a huge difference but it is closer to body in the hierarchy, so I am going to attach to document.

**Comment by jessegreenberg** [comment]
@KatieWoe can you please confirm this is fixed in https://phet-dev.colorado.edu/html/molecules-and-light/1.6.0-dev.1/phet/xhtml/molecules-and-light_all.xhtml?  If so I will merge change with release branch.

**Comment by KatieWoe** [comment]
Looks good on Win 10 Chrome and Mac 10.15 Safari

**Comment by jessegreenberg** [comment]
Great, thanks @KatieWoe

**Comment by jessegreenberg** [comment]
cherry-pick complete, dependencies updated in https://github.com/phetsims/molecules-and-light/commit/e39683fcc5ddbdeecce61e528351bce70f07ff81

**Comment by KatieWoe** [comment]
This is working in https://github.com/phetsims/QA/issues/520 on ChromeOS. Closing
